routes/users.js

```diff
@@ -387,4 +392,119 @@ router.get('/choose', redirectLogin, async (req, res) => {
   }
 });
 
+
+// @desc    Render profile.html, grabs userId and render ejs data in static template
+// @route   GET '/users/choose'
+// @access  Private
+// @tested
+router.get('/password/request', async (req, res) => {
+  try {
+    res.render('requestPassword')
+
+  } catch (err) {
+    return handleError(res, 400, err);
+  }
+});
+
+
+// @desc    Render profile.html, grabs userId and render ejs data in static template
+// @route   GET '/users/choose'
+// @access  Private
+// @tested
+router.post('/password/request', async (req, res) => {
+  try {
+
+    if (!req.body.email) return handleError(res, 400, 'email missing');
+
+    const userObject = await User.findOne({ email: req.body.email });
+
+    if (!userObject) return handleError(res, 400, 'user not found');
```

Comment:
no need to return the handleError, it sends a response inside of it

Reply:
if I dont return the rest of the code is getting called though?